docs/separate-benchmark-source-set.md

```diff
@@ -107,6 +107,6 @@ Set up a separate benchmark source set by following these simple steps:
 4. **Add Benchmarks**
 
    Place your benchmark code into the benchmark source set.
-   Refer to our [writing benchmarks guide](docs/writing-benchmarks.md) for a comprehensive guide on writing benchmarks.
+   Refer to our [writing benchmarks guide](/docs/writing-benchmarks.md) for a comprehensive guide on writing benchmarks.
```

Comment:
Given that in other guide documents we reference the relative location, I would suggest:
 ```suggestion
    Refer to our [writing benchmarks guide](writing-benchmarks.md) for a comprehensive guide on writing benchmarks.
```

Reply:
Could you please fix the same problem in line number 57?

Reply:
@qurbonzoda Absolutely

Reply:
@qurbonzoda Done!

Reply:
https://github.com/Kotlin/kotlinx-benchmark/pull/192#discussion_r1526261349

What do you think about this suggestion?

Reply:
@qurbonzoda Hey, sorry. Didn't fully understood the suggestion. Let me check out overall convention used in the repository for doc links. For sure the most important thing is to keep it all consistent.

Reply:
Totally agree. Relative path is better for doc consistency. Updating.